README.md

```diff
@@ -81,6 +81,12 @@ The fastest and easiest way to try Hyperswitch is via our CDK scripts
 
 3. Follow the instructions provided on the console to successfully deploy Hyperswitch
 
+### One-click deployment on Elestio
+
+Elestio provides option to deploy fully managed option of Huperswitch. Elestio manages the migrations, backups, version changes and security of the application.
```

Comment:
Nit: Typo.
```suggestion
Elestio provides option to deploy fully managed option of Hyperswitch. Elestio manages the migrations, backups, version changes and security of the application.
```

Apart from that, I have a couple of questions:

1. By "migrations", I believe you mean migrating data/services from other cloud providers, correct?
2. How does Elestio handle Hyperswitch configuration changes with new releases of Hyperswitch, during version upgrades?
3. How does Elestio handle database schema changes during version upgrades? The concern here is that some of our database changes need to be done before the version upgrade, and some after, so manual effort is involved most of the time.

I went through your docs, but couldn't find answers for 2 and 3. Would appreciate it if you could clarify them for me.

Reply:
1. Yes, migration means moving the deployed service from other cloud providers to our environment.
2. We're using the docker standalone tag, and we have an automated system that updates the services every Sunday so if is there any changes that need to be required in service after new updates we patch the instances auto or manually depending upon the changes.
3. Currently, we're simply updating the new release of the same tag if any, does your app not support the auto migrations while upgrades or if you have migrations guidelines please share them so we can adopt them for the version updates?

You can check our repo from here [https://github.com/elestio-examples/hyperswitch/blob/main/docker-compose.yml](https://github.com/elestio-examples/hyperswitch/blob/main/docker-compose.yml)

Reply:
> [...] does your app not support the auto migrations while upgrades or if you have migrations guidelines please share them so we can adopt them for the version updates?

Nope, our app server does not natively handle running database migrations. We don't have any specific guidelines for running database migrations, we manually review database migrations before performing version upgrades on our hosted environments, and run them once we've confirmed that they won't break anything (specifically backwards compatibility of our app server), and then proceed to deploy a newer version of the app server.

That said, I did go through your repository. If your system/tool that performs the weekly updates manages to run the `migration_runner` container (in the [`docker-compose.yml` file](https://github.com/elestio-examples/hyperswitch/blob/485f3d703124e8196ff86dd3c9d658828129eb62/docker-compose.yml)) at the time of version upgrades, automated running of database migrations should be handled decently well. My concern with this automated way of running database migrations is that some destructive queries (say those dropping columns or entire tables) may end up being run automatically, thus preventing users from rolling back to an older version of the application (because we no longer have the data for the deleted columns or tables).

Reply:
Hey sorry for late response. We have backups of the entire application data including DB so in case of the updates if any error occurs then will simply restore the data from the backup. Our user can also restore the backup from their side as well from our dashboard.